requirements.txt

```diff
@@ -1,12 +1,19 @@
 Cython>=0.16
 Distutils2==1.0a4
 argparse==1.2.1
-numpy>=1.6.2
-pandas>=0.8.1
-pymc>=2.3.3
+h5py==2.10.0
+numpy>=1.20.1
+scipy>=1.6.1
+pandas>=1.0.0
+pymc>=2.3.8
 python-dateutil==1.5
 pytz==2012d
 wsgiref==0.1.2
 matplotlib>=1.1.0
 git+https://github.com/hddm-devs/kabuki.git@master
-patsy
+seaborn>=0.11.0 
+statsmodels>=0.12.0
+tensorflow==1.15
```

Comment:
Does it need to be this specific?

Reply:
tensorflow is fixed at 1.15 because of some issues regarding performance when switching to tf 2.x. 
Recently an update h5py also affected saved model parsing (which was sort of fixed with newest tf 2.5).
To relax the req, we need to do some more testing to make ensure it's working regardless of version.

seaborn and statsmodels I can probably drop the version requirement to earlier if that's desirable.